pkg/operation/botanist/controlplane.go

```diff
@@ -600,3 +594,79 @@ func (b *Botanist) DeleteCertBroker() error {
 	}
 	return nil
 }
+
+// CreateDeletionCheckpoint creates an empty configmap used as a checkpoint for shoot deletion. It gets created after the Kubernetes resources
+// inside a shoot have been cleaned up and the kube-apiserver is deleted.
+func (b *Botanist) CreateDeletionCheckpoint() error {
+	_, err := b.K8sSeedClient.CreateConfigMap(b.Shoot.SeedNamespace, common.DeletionCheckpoint, nil, true)
```

Comment:
Maybe use the non-deprecated runtime client to create the CM?

Reply:
See above. I don't fully trust the controller-runtime yet. Let's rather completely switch to the controller-runtime at a later point in time

Reply:
I disagree. Where do those 'trust issues' come from? If we have problems, we should address them directly instead of using deprecated functions.